src/main/java/com/onarandombox/MultiverseCore/commands/VersionCommand.java

```diff
@@ -35,94 +39,89 @@
+                "| --- | --- |" + System.lineSeparator() +
+                "| Multiverse-Core Version | `" + this.plugin.getDescription().getVersion() + "` |" + System.lineSeparator() +
+                "| Bukkit Version | `" + this.plugin.getServer().getVersion() + "` |" + System.lineSeparator() +
+                //"| Loaded Worlds | `" + this.plugin.getMVWorldManager().getMVWorlds() + "` |" + System.lineSeparator() +
+                "| Multiverse Plugins Loaded | `" + this.plugin.getPluginCount() + "` |" + System.lineSeparator() +
+                "| Economy being used | `" + plugin.getEconomist().getEconomyName() + "` |" + System.lineSeparator() +
+                "| Permissions Plugin | `" + this.plugin.getMVPerms().getType() + "` |" + System.lineSeparator() +
+                "## Parsed Config" + System.lineSeparator() +
+                "These are what Multiverse thought the in-memory values of the config were." + System.lineSeparator() + System.lineSeparator() +
+                "| Config Key  | Value |" + System.lineSeparator() +
+                "| --- | --- |" + System.lineSeparator() +
+                "| version | `" + this.plugin.getMVConfig().getVersion() + "` |" + System.lineSeparator() +
+                "| messagecooldown | `" + plugin.getMessaging().getCooldown() + "` |" + System.lineSeparator() +
+                "| teleportcooldown | `" + plugin.getMVConfig().getTeleportCooldown() + "` |" + System.lineSeparator() +
+                "| worldnameprefix | `" + plugin.getMVConfig().getPrefixChat() + "` |" + System.lineSeparator() +
+                "| worldnameprefixFormat | `" + plugin.getMVConfig().getPrefixChatFormat() + "` |" + System.lineSeparator() +
+                "| enforceaccess | `" + plugin.getMVConfig().getEnforceAccess() + "` |" + System.lineSeparator() +
+                "| displaypermerrors | `" + plugin.getMVConfig().getDisplayPermErrors() + "` |" + System.lineSeparator() +
+                "| teleportintercept | `" + plugin.getMVConfig().getTeleportIntercept() + "` |" + System.lineSeparator() +
+                "| firstspawnoverride | `" + plugin.getMVConfig().getFirstSpawnOverride() + "` |" + System.lineSeparator() +
+                "| firstspawnworld | `" + plugin.getMVConfig().getFirstSpawnWorld() + "` |" + System.lineSeparator() +
+                "| debug | `" + plugin.getMVConfig().getGlobalDebug() + "` |" + System.lineSeparator();
     }
 
     private String readFile(final String filename) {
-        String result;
+        StringBuilder result;
         try {
             FileReader reader = new FileReader(filename);
             BufferedReader bufferedReader = new BufferedReader(reader);
             String line;
-            result = "";
+            result = new StringBuilder();
             while ((line = bufferedReader.readLine()) != null) {
-                result += line + '\n';
+                result.append(line).append(System.lineSeparator());
```

Comment:
If this is the data going to the paste service I would think `\n` would be appropriate, no?

Reply:
In this case, yes, I don't see anything wrong with `\n`. I was mainly focused on switching to the system's line ending because the info is also printed to the console/logs, but since this is only used for pasting, it might be better to stick with `\n`.